Ansible/templates/CentOS-Base.repo.j2

```diff
@@ -26,3 +26,11 @@ baseurl={{ os_repo }}/centos/$releasever/updates/$basearch/
 gpgcheck=0
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-$releasever
 enabled = {{ update_repo_enabled | default(1) }}
+
+[extras]
+name=CentOS-$releasever - Extras
+mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=extras
```

Comment:
@Pearl1594 do we ned at least this if el7 tries to install haveged and other pkgs?

Reply:
isn't required